pkgs/percona-xtrabackup/8_1.nix

```
@@ -0,0 +1,55 @@
+{ lib, stdenv, fetchurl, bison, boost, cmake, makeWrapper, pkg-config
+, curl, cyrus_sasl, libaio, libedit, libev, libevent, libgcrypt, libgpg-error, lz4
+, ncurses, numactl, openssl, procps, protobuf, valgrind, xxd, zlib
+, perlPackages
+, fetchSubmodules ? false, extraPatches ? [], extraPostInstall ? "", ...
+}:
+
+stdenv.mkDerivation rec {
```

```suggestion
stdenv.mkDerivation (finalAttrs: {
```

Modern nixpkgs prefer this overlay-style way of referring to internal attributes, as this composes much better with overriding these attributes from the outside later.